pulsar-websocket/src/main/java/org/apache/pulsar/websocket/ProducerHandler.java

```diff
@@ -206,7 +242,7 @@ public void onWebSocketText(String message) {
             }
         }).exceptionally(exception -> {
             log.warn("[{}] Error occurred while producer handler was sending msg from {}: {}", producer.getTopic(),
-                    getRemote().getInetSocketAddress().toString(), exception.getMessage());
+                    getRemote().getInetSocketAddress().toString(), exception.getMessage(), exception);
```

Comment:
If we wanna print the exception stack trace, it's better remove the duplicated exception message?

Reply:
fixed